AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -80,16 +70,16 @@ conda activate user_tensorflow
 
 #### Navigate to Model Zoo
 
-Navigate to the Model Zoo for Intel® Architecture source directory. By default, it is in your installation path, like `/opt/intel/oneapi/modelzoo`. 
+Navigate to the Intel® AI Reference models source directory. By default, it is in your installation path, like `/opt/intel/oneapi/modelzoo`. 
 
-1. View the available Model Zoo release versions for the AI Kit:
+1. View the available Intel® AI Reference models release versions for the AI Tools:
    ```
-   ls /opt/intel/oneapi/modelzoo
-   2.11.0  latest
+   ls /opt/intel/oneapi/reference_models
+   2.13.0  latest
    ```
-2. Navigate to the [Model Zoo Scripts](https://github.com/IntelAI/models/tree/v2.11.0/benchmarks) GitHub repo to determine the preferred released version to run inference for ResNet50 or another supported topology.
+2. Navigate to the [Intel® AI Reference models Scripts](https://github.com/IntelAI/models/tree/v2.11.0/benchmarks) GitHub repo to determine the preferred released version to run inference for ResNet50 or another supported topology.
    ```
-   cd /opt/intel/oneapi/modelzoo/latest
+   cd /opt/intel/oneapi/reference_models/latest
    ```
 
 #### Install Jupyter Notebook
```

Comment:
```suggestion
### Run the Jupyter Notebook
```

Reply:
the session indeed is related to install notebook.